app/services/jwt_token_validate.rb

```diff
@@ -0,0 +1,71 @@
+require 'jwt'
+require 'rest-client'
+
+class JwtTokenValidate
+  attr_reader :decoded_token
+  def initialize(jwt_token, url = nil, client_id =nil)
+    @jwt_token = jwt_token
+    @url = url
+    @client_id = client_id
+  end
+
```

Comment:
You can use `delegate :logger, to: :Rails` so you don't have to call `Rails.logger` all the time.

Reply:
thanks :)